src/service/default.service.js

```diff
@@ -11,10 +11,14 @@ let fakeData = {
     }
 };
 
-var Service = function() {  
+var Service = function(window) {  
     this.apiUrl = undefined;
+    this.user = undefined;
     if (typeof window !== 'undefined') {
-        this.apiUrl = window.location.origin;
+        this.apiUrl = window.location.origin;        
```

Comment:
ok now I see why you're passing the window... but is there a way we can save this in app state instead of passing around DOM stuff?